internal/driver/driver.go

```diff
@@ -549,10 +589,11 @@ func (d *Driver) newDevice(name string, protocols map[string]models.ProtocolProp
 			fmt.Sprintf("profile %s not found in core metadata", name), err)
 	}
 
-	fdPath, edgexErr := d.getProtocolProperty(protocols, UsbProtocol, Path)
+	paths, edgexErr := d.getPaths(protocols)
 	if edgexErr != nil {
 		return nil, errors.NewCommonEdgeXWrapper(edgexErr)
 	}
+	fdPath := paths[0].(string)
```

Comment:
i was thinking, would it make sense to add a helper receiver function, `func (dev *Device) getFirstPath()`?